ledger-test-tool/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TransactionServiceCorrectnessIT.scala

```diff
@@ -337,10 +337,21 @@ object TransactionServiceCorrectnessIT {
           events = t.events
             .map(e => e.copy(event = stripEventId(e.event)).modifyWitnessParties(_.sorted)),
           transactionId = "transactionId",
+          traceContext = t.traceContext.map(tc => tc.copy(tc.traceparent.map(eraseSpanId))),
         )
       )
   }
 
+  // Erase span id from the trace context. It is an element of the race context that
+  // is different on the different participants that are handling the transacion stream
+  // requests.
+  def eraseSpanId(parentTraceId: String): String =
+    parentTraceId.split("-").toList match {
+      case ver :: traceId :: _ :: rest =>
```

Comment:
I did not find out how is the order of traceId, spanId specified. Is there any link to it?

Reply:
added a link to the documentation